### ISSUE TITLE
Deprecation notice

### ISSUE THREAD

**Issue by mattdonnelly** [mattdonnelly/Swifter#360]
**TL;DR** This package will be deprecated in about one month. I'm not going to delete it for now but will no longer be maintaining it officially. 

Hi everyone,

I created this project back when Swift was a very new language, as a way of learning it and sharing ideas on how to deal with APIs using it. Since then, it's grown a lot of in popularity and has been used more people than I had ever imagined, which was been amazing to see.

Unfortunately, I haven't had time to contribute actively to this project for several years. For a long time, I relied on other people who did a far better job at maintaining it than me. However, over time this repo has slipped out of date with Twitter's API and there's been less and less activity here.

As a result, I've decided to officially deprecate this project in about one month from now. This means there will be no future changes to the repo, so on the off chance that anyone is still using this, you should seek out alternatives now. I don't plan to the delete the repo to prevent it suddenly breaking people's code but I will be removing all collaborators with write access and updating the README + repo description to indicate the project is no longer maintained. My goal here is to prevent this from becoming yet another horror story about an inactive developer of a widely used library unintentionally distributing malware.

If anyone has other recommended courses of action or questions please leave them here. Massive thanks to everyone that has contributed and helped maintain this project thus far. Especially @meteochu and @zntfdr

I'll follow up here in a months time once I begin taking the steps to deprecate the project.

**Comment by jianming-work** [comment]
It is a sad news.
Everyone have any other library to recommend?

**Comment by mironal** [comment]
@mattdonnelly Thanks for the great library.

I was a Swifter user. I appreciate the contributors.

@jianming-work 

I have created and am still maintaining a new TwitterAPI library for Swift this year.
It supports v1 and v2 APIs and Swift Concurrency.

Please try it out if you like.

https://github.com/mironal/TwitterAPIKit

**Comment by ramzesenok** [comment]
Thanks Matt, was a pleasure to use the library!

**Comment by tarasis** [comment]
Thank you @mattdonnelly , and also @mironal

**Comment by smhk** [comment]
@mattdonnelly  fantastic work and the correct thing to do now.  Using your code was one of the main pleasures of working with devices the last few years.  As you know for every 1 person who bothers to thank you on here, 1000 folks are huge fans!  Thanks again and cheers.

**Comment by meteochu** [comment]
@mattdonnelly (and @zntfdr), it's been an honour! I got to learn a lot about Swift and the Twitter API through this framework and am always glad to have been able to contribute to it in the time that I did. 

Cheers!

**Comment by lk251** [comment]
> https://github.com/mironal/TwitterAPIKit
@mironal hi! i'm very happy to hear about this, please let me know how i can help with the repo.

**Comment by lk251** [comment]
@mattdonnelly thank you

**Comment by mattdonnelly** [comment]
The steps to deprecate this project have been completed and I'm going to leave this issue open for better visibility. Thanks for the words of support everyone!
